server/controllers/ArticleController.js

```diff
@@ -49,7 +49,7 @@ export default class ArticleController {
    * @param {object} res the response object
    * @returns {object} the article that was created.
    */
-  static createArticle(req, res) {
+  static async createArticle(req, res) {
```

Comment:
Good job finishing the task, @yomigeek. Would you explain, though, why you added `async` before this function since you're not awaiting anything inside it? Thanks. If it is an error, though, which I feel it is because you're not using the `await` keyword inside it, then you can remove the `async` keyword.